www-apps/gitea/gitea-1.7.1.ebuild

```diff
@@ -31,21 +31,31 @@ pkg_setup() {
 src_prepare() {
 	default
 	sed -i -e "s/\"main.Version.*$/\"main.Version=${PV}\"/"\
-		-e "s/-ldflags '-s/-ldflags '/" src/${EGO_PN}/Makefile || die
+		-e "s/-ldflags '-s/-ldflags '/" \
+		-e "s/GOFLAGS := -i -v/GOFLAGS := -v/" \
+		"src/${EGO_PN}/Makefile" || die
+	sed -i -e "s#^RUN_MODE = dev#RUN_MODE = prod#"\
+		-e "s#^LOG_SQL = true#LOG_SQL = false#"\
+		-e "s#^ROOT_PATH =#ROOT_PATH = ${EPREFIX}/var/log/gitea#"\
+		-e "s#^MODE = console#MODE = console, file#"\
+		-e "s#^LEVEL = Trace#LEVEL = Info#"\
+		-e "s#^APP_ID =#;APP_ID =#"\
+		-e "s#^TRUSTED_FACETS =#;TRUSTED_FACETS =#"\
+		"src/${EGO_PN}/custom/conf/app.ini.sample" || die
 }
 
 src_compile() {
-	GOPATH="${WORKDIR}/${P}:$(get_golibdir_gopath)" emake -C src/${EGO_PN} generate
-	TAGS="bindata pam sqlite" LDFLAGS="" CGO_LDFLAGS="" GOPATH="${WORKDIR}/${P}:$(get_golibdir_gopath)" emake -C src/${EGO_PN} build
+	GOPATH="${WORKDIR}/${P}:$(get_golibdir_gopath)" emake -C "src/${EGO_PN}" generate
+	TAGS="bindata pam sqlite" LDFLAGS="" CGO_LDFLAGS="-fno-PIC" GOPATH="${WORKDIR}/${P}:$(get_golibdir_gopath)" emake -C "src/${EGO_PN}" build
```

Comment:
Why -fno-PIC?

Reply:
It was there since the beginning (before the last bump). I wasn't be able to find any reason for its initial inclusion, nor for this removal in the last bump. Perhaps we should continue without it?

Reply:
Unless there is a reason, I think we should not disable PIC.